src/js/_enqueues/admin/common.js

```diff
@@ -1939,7 +1939,7 @@ $document.ready( function() {
 	});
 
 	// Event handler for "Go back" link (if present)
-	$('#go-back').click( function () {
+	$('.go-back').click( function () {
```

Comment:
This is a BC break for anyone who is relying on their link with a `go-back` id to get registered.

Reply:
Hi Timothy, thank you for the comment.

Before this PR, all "Go Back" links used inline event handlers, and there weren't event handlers for neither '#go-back' nor '.go-back', I introduced them.
In the beginning, I used an ID, but then I thought there might be more go-back links on a page, so I changed it to a class.

Relevant diffs:
* [src/wp-admin/comment.php](https://github.com/WordPress/wordpress-develop/pull/551/files/cbc83f419eb4e6b74e1361ad74e7c9e04636a154#diff-8c9ed6b351c85b285dfaae508e58600aR74)
* [src/wp-admin/includes/ms.php](https://github.com/WordPress/wordpress-develop/pull/551/files/cbc83f419eb4e6b74e1361ad74e7c9e04636a154#diff-bd071532f9e62766ba0d8b396dad72aaR49)
* [src/wp-admin/setup-config.php](https://github.com/WordPress/wordpress-develop/pull/551/files/cbc83f419eb4e6b74e1361ad74e7c9e04636a154#diff-54e0119d57a572c9f94f4c802355018dR282)

Reply:
My fault for not reading closer 🤦🏻‍♂️. Apologies for the noise!